Homework_24/proxy/proxy_file_reader_writer.py

```diff
@@ -0,0 +1,37 @@
+from Homework_24.proxy.file_reader import FileReader
+from Homework_24.proxy.file_writer import FileWriter
+
+
+class TxtProxyWriterReader:
+    def __init__(self, file_path):
+        self.__result = ''
+        self.__is_actual = False
+        self.__file_reader = FileReader(file_path)
+        self.__file_writer = FileWriter(file_path)
+
+    def read_file(self):
+        if self.__is_actual:
+            return self.__result
+        else:
+            self.__result = self.__file_reader.read_file()
+            self.__is_actual = True
+            return self.__result
+
+    def write_file(self):
+        self.__result = self.__file_writer.append_to_file('new_data')
```

Comment:
new_data should be pass as a parameter. In the proxy method, we should check if new_data == self.result 
if so we do not write nthg